flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/calc/async/AsyncFunctionRunner.java

```diff
@@ -32,7 +32,7 @@
  */
 public class AsyncFunctionRunner extends RichAsyncFunction<RowData, RowData> {
 
-    private static final long serialVersionUID = -6664660022391632123L;
+    private static final long serialVersionUID = -7198305381139008806L;
```

Comment:
Is this an accidental change? Should we set it to `1L`?

Reply:
I think it was accidental since there's no reason to change this.  But setting it to 1 sounds good.